src/features/metadata-instance-editor/CascadePolicy.scss

```diff
@@ -42,7 +42,7 @@ $cascade-policy-background: #f1e2fd;
     padding: 14px 15px;
 }
 
-.metadata-cascade-toggle {
+.toggle-container.metadata-cascade-toggle {
```

Comment:
a new CSS rule on `.toggle-container` was messing with the cascade display. fixing it here.